ssp

```diff
@@ -8826,6 +8828,22 @@ sub check_for_cpro {
 
 ## END malware checks
 
+sub detect_slowloris {
+    my $sockcount = 75;
+    my $count = 0;
+
+    return unless my $netstat_out = timed_run( 0, 'netstat', '-nap' );
+    for my $line ( split( '\n', $netstat_out ) ) {
+        if ( $line =~ m/ESTABLISHED.*httpd/ ) {
+            $count++;
+        }
+    }
+    if ( $count >= $sockcount ) {
+        print_warn("Possible SLOWLORIS attack: Found ".$count." Apache sockets in ESTABLISHED\n");
+        print_warning("\t \\_ See https://documentation.cpanel.net/display/EA/How+To+Mitigate+Slowloris+Attacks");
+    }
+}
+
```

Comment:
We really would like this to be a more robust check.  We need to assure that the IPS are also unique as SlowLorris tends to always prefer a single IP, correct?

I'm not sure how automatable we can make a slowlorris detection and not constantly throw noise.